features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
+
+Items represent functionality that is used by the application (mainly user interfaces or automation logic). Items have a state and are used through events. They are objects that can be read from, or written to, in order to interact with them.
+Items can be bound to bindings i.e. for reading the status from e.g. Sonos, or for updating them. Read the wiki page for the respective binding for more help and examples.
+Items must also be defined in one or more files in the folder `${openhab_home}/conf/items`. Files here must have the extension .items but you can make as many .items files as you need/want however each item must be unique across them all.
+Groups are also defined in the .items files. Groups can be nested inside other groups, and items can be in none, one, or more groups.
+
+Typically items are defined using the openHAB Designer by editing the items definition files. Doing so you will have full IDE support like syntax checking, context assist etc.
+
+## Item Syntax
+Items are defined in the following syntax:
+```xtend
+itemtype  itemname  ["labeltext"]  [<iconname>]  [(group1, group2, ...)]  [{bindingconfig}]
+```
+Note: Parts in square brackets ([]) are optional.
+
+**Example**
+```xtend
+Number  Sensor_Temperature  "The Temperature is [%.1f °C]"  <temperature>  (gTemperature, gSensor)  {knx="1/0/15+0/0/15"}
+```
+The example above defines an item:
+- of type `Number`
+- with name `Sensor_Temperature`
+- formatting its output in format `%.1f °C`
+- displaying icon `temperature`
+- belonging to groups `gTemperature` and `gSensor`
+- bound to openHAB binding `knx` with write group address `1/0/15` and listening group address `0/0/15`
+
+## Item Types
+The item type defines which kind of values can be stored in that item and which commands can be send to it. Each item type has been optimized for certain components in your smart home. This optimization is reflected in the data types and commands types. Little example for better understanding: Have a look at the following table and find the itemtype "Color". A Philips Hue RGB light bulb provides you three information. Is the bulb on or off, its current brightness and the color. If you want to change one of these values you can use four commands. Switching the bulb on or off, increasing or decreasing the brightness, setting the brightness directly to a specific value and you can change the bulb's color.
+
+OpenHAB2 provides you the following item types (alphabetical order):
```

Comment:
In the sitemap article, I decided to switch from alphabetic to relevance sorting. I think that makes may more sense, as most item types are quite special while others like switch or number should be promoted on the top of the list